tools/run_danesfield.py

```diff
@@ -473,7 +473,7 @@ def main(args):
                  cmd_args)
 
     texture_mapping_outdir = os.path.join(working_dir, 'texture-mapping')
-    occlusion_mesh = "xxxx.obj"
+    occlusion_mesh = os.path.join(roof_geon_extraction_outdir, 'occlusion_mesh.obj')
```

Comment:
I don't recall exactly the purpose / origin "xxxx.obj" file, but I don't know that this was a typo.  Are these two files by chance the same, one just has a much more descriptive name?

Reply:
The legacy code would output xxxx.obj to the working directory, and with such name it was hard to tell what the file was and its purpose. The modification ensures that all intermediate and final output gets where it belongs, i.e. respective output directory.

Reply:
Ah I see, this is setting the output path for the occlusion mask.  Totally on board with this change then, thanks.